test/static_test.dart

```diff
@@ -13,102 +13,100 @@ import 'common.dart' as common show main;
-      smoke_0.L2: smoke_0.L,
-      _M0: smoke_0.C,
-    },
-    declarations: {
-      smoke_0.A: {
-        #i: const Declaration(#i, int),
-        #inc0: const Declaration(#inc0, Function, kind: METHOD),
-        #inc1: const Declaration(#inc1, Function, kind: METHOD),
-        #inc2: const Declaration(#inc2, Function, kind: METHOD),
-        #j: const Declaration(#j, int),
-        #j2: const Declaration(#j2, int, kind: PROPERTY),
-      },
-      smoke_0.B: {
-        #a: const Declaration(#a, smoke_0.A),
-        #f: const Declaration(#f, int, isFinal: true),
-        #w: const Declaration(#w, int, kind: PROPERTY),
-      },
-      smoke_0.C: {
-        #b: const Declaration(#b, smoke_0.B),
-        #inc: const Declaration(#inc, Function, kind: METHOD),
-        #x: const Declaration(#x, int),
-        #y: const Declaration(#y, String),
-      },
-      smoke_0.D: {
-        #i2: const Declaration(#i2, int, kind: PROPERTY, isFinal: true),
-        #x2: const Declaration(#x2, int, kind: PROPERTY, isFinal: true),
-      },
-      smoke_0.E: {
-        #noSuchMethod: const Declaration(#noSuchMethod, Function, kind: METHOD),
-        #y: const Declaration(#y, int, kind: PROPERTY, isFinal: true),
-      },
-      smoke_0.E2: {},
-      smoke_0.F: {
-        #staticMethod: const Declaration(#staticMethod, Function, kind: METHOD, isStatic: true),
-      },
-      smoke_0.F2: {},
-      smoke_0.G: {
-        #b: const Declaration(#b, int, annotations: const [smoke_0.a1]),
-        #d: const Declaration(#d, int, annotations: const [smoke_0.a2]),
-      },
-      smoke_0.H: {
-        #f: const Declaration(#f, int, annotations: const [smoke_0.a1]),
-        #g: const Declaration(#g, int, annotations: const [smoke_0.a1]),
-        #h: const Declaration(#h, int, annotations: const [smoke_0.a2]),
-        #i: const Declaration(#i, int, annotations: const [smoke_0.a3]),
-      },
-      smoke_0.K: {
-        #k: const Declaration(#k, int, annotations: const [const smoke_0.AnnotC(named: true)]),
-        #k2: const Declaration(#k2, int, annotations: const [const smoke_0.AnnotC()]),
-      },
-      smoke_0.L: {
-        #incM: const Declaration(#incM, Function, kind: METHOD),
-        #m: const Declaration(#m, int, kind: PROPERTY, isFinal: true),
-      },
-      smoke_0.L2: {
-        #incM: const Declaration(#incM, Function, kind: METHOD),
-        #m: const Declaration(#m, int),
-        #n: const Declaration(#n, int),
-      },
-      _M0: {
-        #i: const Declaration(#i, int),
-        #inc: const Declaration(#inc, Function, kind: METHOD),
-        #inc0: const Declaration(#inc0, Function, kind: METHOD),
-        #j: const Declaration(#j, int),
-        #j2: const Declaration(#j2, int, kind: PROPERTY),
-      },
-    },
-    staticMethods: {
-      smoke_0.A: {
-        #staticInc: smoke_0.A.staticInc,
-      },
-    },
-    names: {
-      #i: r'i',
-    });
+final configuration = new StaticConfiguration(checkedMode: false, getters: {
```

Comment:
you probably need to revert this file. It's autogenerated and I believe a test might be checking that it is up-to-date.